src/common/utility.h

```diff
@@ -68,9 +69,19 @@ OCSYNC_EXPORT Q_DECLARE_LOGGING_CATEGORY(lcUtility)
       * Currently only implemented on Windows.
       */
     OCSYNC_EXPORT bool hasSystemLaunchOnStartup(const QString &appName);
+
+    /**
+     * @brief Return if launch on startup is enabled for the current user.
+     * @param appName the name of the application (because of branding)
+     */
     OCSYNC_EXPORT bool hasLaunchOnStartup(const QString &appName);
     OCSYNC_EXPORT void setLaunchOnStartup(const QString &appName, const QString &guiName, bool launch);
 
+#ifdef Q_OS_MAC
+    /// Migrate launch on startup (only used on macOS).
+    OCSYNC_EXPORT void migrateLaunchOnStartup(const QString &appDomain);
```

Comment:
Can be moved to the mac platform I assume?

Reply:
Will do that